andela/travel_tool_front
# 159393234  Have cursor over the button to be a pointer

#### What does this PR do?
- The PR adds the cursor over the buttons to be pointers.

#### Description of Task to be completed?
Change css styling of the button so that when user hovers over it they can know it is clickable

#### How should this be manually tested?
- Git clone the repository
- Check into the project directory
- Run yarn install
- Run yarn start
- Click login button to access requests page
- Hover over the header buttons and the cursor changes

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#159393234
#### Screenshots (if appropriate)

#### Questions:
None
